app.js

```diff
@@ -19,24 +6,52 @@ window.onload = function() {
   var buttonStart = document.getElementById("button-start");
   var buttonStop = document.getElementById("button-stop");
   var buttonReset = document.getElementById("button-reset");
+  var cart1 = document.querySelector('#cart1') 
   var Interval;
+  let cart1Left = 0;
```

Comment:
Remove commented out code from file. (This code is preserved by your git commits, you can always go back for it later if you need it so no need to keep it in comments)